src/command_base.ts

```diff
@@ -103,9 +103,9 @@ export abstract class BaseCommand {
 		const member = await guild.members.fetch(user);
 
 		switch (permission) {
-		case 'EVAL':
+			case 'EVAL':
```

Comment:
Honestly switch indenting twice annoys me alot, can we make it so that you don't need to indent case labels?

Reply:
sure.